columnphysics/icepack_brine.F90

```diff
@@ -1000,7 +765,8 @@ end subroutine icepack_init_hbrine
 
 !=======================================================================
 !autodocument_start icepack_init_zsalinity
-!  Initialize zSalinity
+!  **DEPRECATED**, all code removed
+!  Interface provided for backwards compatibility
```

Comment:
I'm not sure I understand what would break if this were removed.  Could the interface be put in an 'ifdef undeprecate' for the next release and then removed completely after that?  Actually, since we had partially deprecated salinity earlier, it was broken and so I think notice has been served -- no one should be calling it at this point.  Is the backwards compatibility issue just so that other (non-CICE) drivers can update icepack without having to make changes in the driver itself?